packages/components/src/Button/ButtonSet.tsx

```diff
@@ -78,7 +81,7 @@ export type ButtonSetType<
   T extends string | string[] = string[]
 > = ForwardRefExoticComponent<ButtonSetProps<T> & { ref: Ref<HTMLDivElement> }>
 
-export const ButtonSet = forwardRef(
+export const ButtonSetInternal = forwardRef(
```

Comment:
```suggestion
export const ButtonSeLayout = forwardRef(
```

`Layout` seems to be winning the war against `Internal` within our code base as-of late (I prefer it since it's more about the the DOM "layout" rather than some secret internal interface. :)